### ISSUE TITLE
Some UI Suggestions, Title and Summary

### ISSUE THREAD

**Issue by bakinazik** [prof18/feed-flow#252]
article title should not be the same thickness as the summary (this reduces readability) and the summary should be hideable.

thank you so much for develop this app.

**Comment by Buxuna** [comment]
If the read articles are slightly dimmer that would be a nice addition.

**Comment by avagraphique** [comment]
I agree, making the titles in the list bolder and perhaps a little larger will greatly improve readability.
